.vib/redmine/cypress/cypress/integration/redmine_spec.js

```diff
@@ -0,0 +1,86 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows admin to login/logout', () => {
+  cy.login();
+  cy.get('#flash_error').should('not.exist');
```

Comment:
When possible, rather check the error message than the element's id, e.g.:

```
cy.contains('Invalid user').should('not.exist');
```

Reply:
In the case of error messages within consistent UI error elements, I disagree. Especially for login.
Imagine that the login test fails for something **other** than username and password being incorrect.
The error message will be different. 
But, the UI element (flash error) showing the error message will still be the same since it's used consistently in the application.
If we limit ourselves to discovering error messages only, we will catch only one type of issue.
WDTY? 

![Screenshot 2022-05-11 at 11 57 59](https://user-images.githubusercontent.com/47592680/167824123-24476297-8b68-4ad5-a2c6-c7c846607fe1.png)

Reply:
I agree with you here! My concern was that text is less prone to change, but I totally see your point